blueprints/unifi/update.sh

```diff
@@ -2,25 +2,28 @@
 # This file contains the update script for unifi
 # Unifi Controller is updated through pkg, Unifi-Poller is not. This script updates Unifi-Poller
 
-if [[ ! "${!POLLER}" ]]; then
-	echo "Skipping Unifi Poller for update, not installed"
+POLLER="jail_${1}_unifi_poller"
+
+# shellcheck disable=SC2154
+if [ "${!POLLER}" = true ]; then	
```

Comment:
I think we should use string comparison here:
```suggestion
if [ "${!POLLER}" == true ]; then	
```